src/_pytest/fixtures.py

```diff
@@ -792,6 +793,25 @@ def _teardown_yield_fixture(fixturefunc, it):
         )
 
 
+def _eval_scope_callable(callable, fixture_name, config):
```

Comment:
I'd change `callable` -> `scope_callable`, just to avoid shadowing the `callable` builtin.